MAINTAINERS.md

```diff
@@ -0,0 +1,66 @@
+# SPIRE Helm Charts Maintainership Guidelines and Processes
+
+This document captures the values, guidelines, and processes that the SPIRE Helm Charts project and its maintainers adhere to. All maintainers agree to uphold and abide by the text contained herein.
+
+For a list of active SPIRE Helm Charts maintainers, please see the [CODEOWNERS](CODEOWNERS) file.
+
+## General governance
+
+The SPIRE Helm CHarts project abides by the same [governance procedures][1] as the SPIFFE project, and ultimately reports to the [SPIFFE Steering Committee][3] (SSC) the same way that the SPIFFE project and associated maintainers do.
+
+SSC members do not track day-to-day activity in the SPIFFE/SPIRE projects, and this should be considered when deciding to raise issues to them. While the SSC has the ultimate say, in practice they are only engaged upon serious maintainer disagreement.
+
+## Maintainer responsibility
+
+SPIRE maintainers adhere to the [requirements and responsibilities][2] set forth in the SPIFFE governance document. They further pledge the following:
+
+* To act in the best interest of the project at all times.
+* To ensure that project development and direction is a function of community needs.
+* To never take any action while hesitant that it is the right action to take.
+* To fulfill the responsibilities outlined in this document and its dependents.
+
+## Guiding principles
+
+The goal of this project is to deliver Helm Charts to the user that are stable, secure, and simple.
+
+1. **Stability** -- The focus of this project is on stability over rapid feature development. SPIRE is a key infrastructure component that needs to be stable. While it is important to develop new features regularly, maintainers must take care that the project is stable.
```

Comment:
👍

Reply:
As much as I like "Stability", I know from many years of experience there is no stability without adoption and usage. We should promote and make it easy to introduce new features, functions, and components, because that's the only way to make them better. They don't have to be enabled by default. In fact, they shouldn't, but we must make it easy to make them available. This is the best way to grow the community and bring wide participation, new points of view, new use-cases, from which we all benefit.